helper_functions.py

```diff
@@ -0,0 +1,13 @@
+"""
+This module consist of independent helper functions
+"""
+
+
+def print_as_table(headers, rows):
```

Comment:
Nice utility to print as a table. However, it would not occur to me to find it here, based on the filename. Try a more specific name

Reply:
@sudeeprp thanks for review I have given it a try, please have a look !